sdk/python/feast/client.py

```diff
@@ -547,6 +548,25 @@ def get_batch_features(
         entity_rows: Union[pd.DataFrame, str],
         compute_statistics: bool = False,
         project: str = None,
+    ) -> RetrievalJob:
+        """
+        Deprecated. Please see get_historical_features.
+        """
+        warnings.warn(
+            "The method get_batch_features() is being deprecated. Please use the identical get_historical_features(). "
+            "Feast 0.7 and onwards will not support get_batch_features().",
+            DeprecationWarning,
+        )
+        return self.get_historical_features(
+            feature_refs, entity_rows, compute_statistics, project
+        )
+
+    def get_historical_features(
+        self,
+        feature_refs: List[str],
+        entity_rows: Union[pd.DataFrame, str],
+        compute_statistics: bool = False,
+        project: str = None,
```

Comment:
Should also update the example to use `get_historical_features()` instead of `get_batch_features()`.

Reply:
Done.